src/domain/Notification.ts

```diff
@@ -11,28 +12,27 @@ import { makeFullSentNotification } from './GetNotificationDetailRepository';
 
 export type Notification = FullSentNotification & Pick<NotificationRequest, 'notificationRequestId'>;
 
-const makeNotificationFromFind =
-  (notificationRequest: NotificationRequest, sentAt: Date, senderPaId: string) =>
-  (findNotificationRequestRecord: CheckNotificationStatusRecord): O.Option<Notification> =>
+export const make = (notificationRequest: NotificationRequest, sentAt: Date, senderPaId: string, iun: Iun) => ({
```

Comment:
```suggestion
export const makeNotification = (notificationRequest: NotificationRequest, sentAt: Date, senderPaId: string, iun: Iun) => ({
```
or a name that immediately tells what it's doing